--- /etc/init.d/kojid	2014-02-26 14:22:18.000000000 -0500
+++ /etc/init.d/kojid	2014-04-07 16:52:53.252001501 -0400
@@ -33,13 +33,13 @@
     cd /
     ARGS=""
     [ "$FORCE_LOCK" == "Y" ] && ARGS="$ARGS --force-lock"
-    [ "$KOJID_DEBUG" == "Y" ] && ARGS="$ARGS --debug"
-    [ "$KOJID_VERBOSE" == "Y" ] && ARGS="$ARGS --verbose"
-    # XXX Fix for make download-checks in kernel builds
-    # Remove once we're running the buildSRPMFromSCM task
-    # as an unpriviledged user with their own environment
-    export HOME="/root"
-    daemon /usr/sbin/kojid $ARGS
+    [ "$KOJIRA_DEBUG" == "Y" ] && ARGS="$ARGS --debug"
+    [ "$KOJIRA_VERBOSE" == "Y" ] && ARGS="$ARGS --verbose"
+    if [ -n "$RUNAS" -a "$RUNAS" != "root" ]; then
+        daemon --user "$RUNAS" /usr/sbin/kojid $ARGS
+    else
+        daemon /usr/sbin/kojid $ARGS
+    fi
     RETVAL=$?
     echo
     [ $RETVAL -eq 0 ] && touch /var/lock/subsys/kojid
