--- /etc/koji-hub/hub.conf  2014-02-26 14:22:18.000000000 -0500
+++ /etc/koji-hub/hub.conf  2014-04-07 16:17:42.144000015 -0400
@@ -9,8 +9,8 @@
 ## Basic options ##
 DBName = koji
 DBUser = koji
-#DBHost = db.example.com
-#DBPass = example_password
+#DBHost = localhost.localdomain
+#DBPass = koji
 KojiDir = /mnt/koji


@@ -25,7 +25,6 @@
 ## end Kerberos auth configuration


-
 ##  SSL client certificate auth configuration  ##
 #note: ssl auth may also require editing the httpd config (conf.d/kojihub.conf)

@@ -33,25 +32,25 @@
 # DNUsernameComponent = CN
 ## separate multiple DNs with |
 # ProxyDNs = /C=US/ST=Massachusetts/O=Example Org/OU=Example User/CN=example/emailAddress=example@example.com
+ProxyDNs = /C=XX/ST=SomeState/L=SomeCity/O=SomeOrganization/OU=SomeOrganizationalUnit/CN=kojiweb/emailAddress=koji@localdomain

 ## end SSL client certificate auth configuration


-
 ##  Other options  ##
 LoginCreatesUser = On
-KojiWebURL = http://kojiweb.example.com/koji
+KojiWebURL = http://kojiweb.localdomain/koji
 # The domain name that will be appended to Koji usernames
 # when creating email notifications
 #EmailDomain = example.com
 # whether to send the task owner and package owner email or not on success.  this still goes to watchers
 NotifyOnSuccess = True
 ## Disables all notifications
-# DisableNotifications = False
+DisableNotifications = True

 ## Extended features
 ## Support Maven builds
-# EnableMaven = False
+EnableMaven = True
 ## Support Windows builds
 # EnableWin = False

@@ -82,4 +81,3 @@
 ## If ServerOffline is True, the server will always report a ServerOffline fault (with
 ## OfflineMessage as the fault string).
 ## If LockOut is True, the server will report a ServerOffline fault for all non-admin
-## requests.
