--- /etc/koji.conf	2014-02-26 14:22:18.000000000 -0500
+++ /etc/koji.conf	2014-04-07 16:17:42.063000014 -0400
@@ -3,16 +3,16 @@
 ;configuration for koji cli tool
 
 ;url of XMLRPC server
-server = http://koji.fedoraproject.org/kojihub
+server = http://koji.localdomain/kojihub
 
 ;url of web interface
-weburl = http://koji.fedoraproject.org/koji
+weburl = http://koji.localdomain/koji
 
 ;url of package download site
-topurl = http://koji.fedoraproject.org/
+topurl=http://koji.localdomain/kojifiles
 
 ;path to the koji top directory
-topdir = /mnt/koji
+;topdir = /mnt/koji
 
 ;configuration for Kerberos authentication
 
@@ -22,10 +22,10 @@
 ;configuration for SSL authentication
 
 ;client certificate
-cert = ~/.fedora.cert
+cert = ~/.koji/client.crt
 
 ;certificate of the CA that issued the client certificate
-ca = ~/.fedora-server-ca.cert
+ca = ~/.koji/serverca.crt
 
 ;certificate of the CA that issued the HTTP server certificate
-serverca = ~/.fedora-server-ca.cert
+serverca = ~/.koji/serverca.crt
