--- /etc/httpd/conf.d/kojiweb.conf	2014-04-07 16:17:42.096000014 -0400
+++ /etc/httpd/conf.d/kojiweb.conf	2014-04-07 16:17:42.106000014 -0400
@@ -1,4 +1,4 @@
-#We use wsgi by default
+# We use wsgi by default
 Alias /koji "/usr/share/koji-web/scripts/wsgi_publisher.py"
 #(configuration goes in /etc/kojiweb/web.conf)
 
@@ -7,6 +7,7 @@
     SetHandler wsgi-script
     Order allow,deny
     Allow from all
+    # Require all granted # Apache 2.4 Compatibility
 </Directory>
 
 # Support for mod_python is DEPRECATED. If you still need mod_python support,
@@ -40,11 +41,11 @@
 # </Location>
 
 # uncomment this to enable authentication via SSL client certificates
-# <Location /koji/login>
-#     SSLVerifyClient require
-#     SSLVerifyDepth  10
-#     SSLOptions +StdEnvVars
-# </Location>
+<Location /koji/login>
+    SSLVerifyClient require
+    SSLVerifyDepth  10
+    SSLOptions +StdEnvVars
+</Location>
 
 Alias /koji-static/ "/usr/share/koji-web/static/"
 
@@ -53,4 +54,5 @@
     AllowOverride None
     Order allow,deny
     Allow from all
+    # Require all granted # Apache 2.4 Compatibility
 </Directory>
