--- /var/lib/pgsql/data/pg_hba.conf	2014-04-07 16:19:25.352000001 -0400
+++ /var/lib/pgsql/data/pg_hba.conf	2014-04-07 16:41:16.005000025 -0400
@@ -66,9 +66,20 @@
 
 # TYPE  DATABASE    USER        CIDR-ADDRESS          METHOD
 
+# Koji
+local   koji        koji                                         trust
+local   koji        apache                                       trust
+host    koji        koji                127.0.0.1/32             trust
+host    koji        apache              127.0.0.1/32             trust
+host    koji        koji                ::1/128                  trust
+host    koji        apache              ::1/128                  trust
+host    koji        koji                192.168.122.2/32         trust
+host    koji        apache              192.168.122.2/32         trust
+
 # "local" is for Unix domain socket connections only
 local   all         all                               ident
 # IPv4 local connections:
 host    all         all         127.0.0.1/32          ident
 # IPv6 local connections:
 host    all         all         ::1/128               ident
+
