--- /var/lib/pgsql/data/pg_hba.conf     2014-04-08 14:20:06.778636025 +0200
+++ /var/lib/pgsql/data/pg_hba.conf     2014-04-08 14:52:01.499379404 +0200
@@ -66,6 +66,16 @@

 # TYPE  DATABASE    USER        CIDR-ADDRESS          METHOD

+# Koji
+local   koji        koji                                         trust
+local   koji        apache                                       trust
+host    koji        koji                127.0.0.1/32             trust
+host    koji        apache              127.0.0.1/32             trust
+host    koji        koji                ::1/128                  trust
+host    koji        apache              ::1/128                  trust
+host    koji        koji                192.168.122.2/32         trust
+host    koji        apache              192.168.122.2/32         trust
+
 # "local" is for Unix domain socket connections only
 local   all         all                               ident
 # IPv4 local connections:
