*** /var/lib/pgsql/data/postgresql.conf.orig    2014-04-09 20:06:42.292842580 +0200
--- /var/lib/pgsql/data/postgresql.conf 2014-04-09 20:13:47.921308706 +0200
***************
*** 56,62 ****

  # - Connection Settings -

! #listen_addresses = 'localhost'               # what IP address(es) to listen on;
                                        # comma-separated list of addresses;
                                        # defaults to 'localhost', '*' = all
                                        # (change requires restart)
--- 56,62 ----

  # - Connection Settings -

! #listen_addresses = '*'                       # what IP address(es) to listen on;
                                        # comma-separated list of addresses;
                                        # defaults to 'localhost', '*' = all
                                        # (change requires restart)
