--- /etc/kojiweb/web.conf	2014-04-07 16:17:42.177000014 -0400
+++ /etc/kojiweb/web.conf	2014-04-07 16:17:42.185000014 -0400
@@ -3,20 +3,18 @@
 #KojiTheme = mytheme
 
 # Key urls
-KojiHubURL = http://hub.example.com/kojihub
-KojiFilesURL = http://server.example.com/kojifiles
+KojiHubURL = http://koji.localdomain/kojihub
+KojiFilesURL = http://koji.localdomain/kojifiles
 
 # Kerberos authentication options
 # WebPrincipal = koji/web@EXAMPLE.COM
 # WebKeytab = /etc/httpd.keytab
 # WebCCache = /var/tmp/kojiweb.ccache
-# The service name of the principal being used by the hub
-# KrbService = host
 
 # SSL authentication options
-# WebCert = /etc/kojiweb/kojiweb.crt
-# ClientCA = /etc/kojiweb/clientca.crt
-# KojiHubCA = /etc/kojiweb/kojihubca.crt
+WebCert = /etc/pki/koji/certs/kojiweb.crt
+ClientCA = /etc/pki/koji/certs/clientca.crt
+KojiHubCA = /etc/pki/koji/certs/kojihubca.crt
 
 LoginTimeout = 72
 
