diff --git a/launchers/base/src/main/resources/bin/init/aprox b/launchers/base/src/main/resources/bin/init/aprox
index 7bf8b75..3cb2a5a 100755
--- a/bin/init/aprox
+++ b/bin/init/aprox
@@ -23,26 +23,36 @@
 
 PROG=aprox
 APROX_HOME=/opt/aprox
-PID=$(ps -C "java -jar $APROX_HOME/lib/aprox-cdi-components-0.14.6.jar" -o pid h)
 LOCKFILE=/var/lock/subsys/$PROG
 EXEC=$APROX_HOME/bin/aprox.sh
 RETVAL=0
 
 start() {
-    echo -n $"Starting $PROG: "
-    $EXEC > /dev/null 2>&1 &
-    [ $RETVAL -eq 0 ] && success || failure
-    echo
-    [ $RETVAL -eq 0 ] && touch /var/lock/subsys/$PROG
+    PID=$(ps ax | grep -v grep | grep "$APROX_HOME/lib/aprox-cdi-components-0.14.6.jar" | grep java | awk '{print $1}' | sort | uniq)
+    if [ "x$PID" == "x" ]; then
+      echo -n $"Starting $PROG: "
+      $EXEC > /dev/null 2>&1 &
+      [ $RETVAL -eq 0 ] && success || failure
+      echo
+      [ $RETVAL -eq 0 ] && touch /var/lock/subsys/$PROG
+    else
+      echo "aprox (pid ${PID}) already running"
+      REtVAL=0
+    fi
 }
 stop() {
+    PID=$(ps ax | grep -v grep | grep "$APROX_HOME/lib/aprox-cdi-components-0.14.6.jar" | grep java | awk '{print $1}' | sort | uniq)
     echo -n $"Shutting down $PROG: "
 
     if [ "x$PID" == "x" ]; then
       echo "not running"
       RETVAL=0
     else
-      killproc ${PROG} > /dev/null 2>&1
+      echo ""
+      for pid in ${PID}; do
+        echo "Stopping $pid"
+        /bin/kill -15 ${pid}
+      done
       RETVAL=$?
       [ $RETVAL -eq 0 ] && success || failure
       echo
@@ -51,6 +61,8 @@ stop() {
 }
 restart() {
     stop
+    echo "Waiting 10s for old process to stop"
+    sleep 10
     start
 }
 
@@ -74,6 +86,7 @@ case "$1" in
         then
             echo "$PROG is stopped"
         else
+            PID=$(ps ax | grep -v grep | grep "$APROX_HOME/lib/aprox-cdi-components-0.14.6.jar" | grep java | awk '{print $1}' | sort | uniq)
             echo "$PROG (pid  ${PID}) is running..."
         fi
     ;;
