diff --git a/launchers/base/src/main/resources/bin/init/aprox b/launchers/base/src/main/resources/bin/init/aprox
index 7bf8b75..b101073 100755
--- a/bin/init/aprox
+++ b/bin/init/aprox
@@ -23,7 +23,7 @@
 
 PROG=aprox
 APROX_HOME=/opt/aprox
-PID=$(ps -C "java -jar $APROX_HOME/lib/aprox-cdi-components-0.14.6.jar" -o pid h)
+PID=$(ps ax | grep -v grep | grep "$APROX_HOME/lib/aprox-cdi-components-0.14.6.jar" | grep java | awk '{print $2}' | sort | uniq)
 LOCKFILE=/var/lock/subsys/$PROG
 EXEC=$APROX_HOME/bin/aprox.sh
 RETVAL=0
@@ -42,7 +42,11 @@ stop() {
       echo "not running"
       RETVAL=0
     else
-      killproc ${PROG} > /dev/null 2>&1
+      echo ""
+      for pid in ${PID}; do
+        echo "Stopping $pid"
+        /bin/kill -15 ${pid}
+      done
       RETVAL=$?
       [ $RETVAL -eq 0 ] && success || failure
       echo
