diff --git a/addons/dot-maven/common/src/main/data/templates/settings.xml.groovy b/addons/dot-maven/common/src/main/data/templates/settings.xml.groovy
index 1f530b3..cbdb62f 100644
--- a/var/lib/aprox/data/templates/settings.xml.groovy
+++ b/var/lib/aprox/data/templates/settings.xml.groovy
@@ -10,7 +10,7 @@
       Red Hat, Inc. - initial API and implementation
 -->
 <settings>
-  <localRepository>${user.home}/.m2/repo-${type}-${name}</localRepository>
+  <localRepository>${System.getProperty('user.home')}/.m2/repo-${type}-${name}</localRepository>
   <mirrors>
     <mirror>
       <id>${name}</id>

